charts/fpl-case-service/values.preview.template.yaml

```diff
@@ -30,6 +30,7 @@ ccd:
   ingressHost: ${SERVICE_FQDN}
   ingressIP: ${INGRESS_IP}
   consulIP: ${CONSUL_LB_IP}
+  cpuRequests: 250m
```

Comment:
@akomisarek why is that needed here since pipeline sets `--set global.devMode=true`?

Reply:
I think the ccd-charts templates are not using the devcpu. 

After my change the CCD pod is getting:
    Restart Count:  0
    Limits:
      cpu:     2500m
      memory:  1Gi
    Requests:
      cpu:      250m
      memory:   512Mi